solutions/DataFramesKoans.jl

```diff
@@ -0,0 +1,53 @@
+module DataFramesKoans
+
+#=
+Links:
```

Comment:
@AminArria Can you move the documentation from the solutions file to the koan? That way people can get a hint without looking at the solution

Reply:
Done in 7fb9424